### ISSUE TITLE
html editor not detecting content on copy/paste from external sources such as MS Word

### ISSUE THREAD

**Issue by jgautrais** [aimeos/ai-admin-jqadm#231]
**Environment**
1. Version 2021.10
2. Integration Laravel 8.68.1
3. Docker with php 7.4

**Describe the bug**
The html editor does not recognize content when copy/pasting from MS Word. We noticed this bug in production.
This is due to content from MS Word having a specific class attribute:
![aimeos-htmleditor-msword](https://user-images.githubusercontent.com/29978924/167649551-efe5a71e-bc62-49c2-a0bb-a00bad302962.png)
The console is throwing the following error:
> Uncaught TypeError: Cannot read properties of null (reading 'length')

The code throwing the error is the following (in [ai-admin-jqadm/js/components/html-editor.js](https://github.com/aimeos/ai-admin-jqadm/blob/666e388c4cab2352bc14b466f0b413c4cc0cf88e/js/components/html-editor.js), line 50):
```
if(this.content.match(/<p>/g).length === 1 && this.content.startsWith('<p>') && this.content.endsWith('</p>')) {
	...
}
```
The regex `/<p>/g` tested in the match function does not handle p tag having a class attribute and the match function is returning `null` if no matches are found ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)).
Therefore `this.content.match(/<p>/g).length` is throwing an error.

**Proposed solution**
Locally we can solve the issue by checking first that `this.content.match(/<p>/g)` is thruthy

**Comment by AurelGit** [comment]
Same trouble, if no `<p>` tag exists, it will raise an error and the editor do not consider that the text is modified.

**Comment by aimeos** [comment]
Fixed in dev-master 2022.04.x-dev and 2021.10.x-dev. New versions will be tagged this week.
